taccsite_cms/urls.py

```diff
@@ -6,42 +6,37 @@
 from django.conf.urls.static import static
 from django.conf.urls import include, url
 from django.contrib import admin
+from django.contrib.auth import views
 from django.contrib.sitemaps.views import sitemap
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 from django.views.static import serve
+from taccsite_cms import remote_cms_auth as remote_cms_auth
 
 from django.http import request
 from django.views.generic.base import RedirectView
-
 admin.autodiscover()
 
 urlpatterns = [
     url(r'^sitemap\.xml$', sitemap,
         {'sitemaps': {'cmspages': CMSSitemap}}),
 
     url(r'^admin/', admin.site.urls),  # NOQA
+    url(r'^cms/logout/', views.LogoutView.as_view(), name='logout'),
 ]
 
-if settings.PORTAL:
+if getattr(settings, 'INCLUDES_CORE_PORTAL', True):
     from django.views.generic.base import TemplateView
     urlpatterns += [
         # FAQ: Allow direct access to markup for Portal and User Guide to render
         url(r'^cms/nav/search/markup/$', TemplateView.as_view(template_name='nav_search.raw.html'), name='search_bar_markup'),
         url(r'^cms/nav/pages/markup/$', TemplateView.as_view(template_name='nav_cms.raw.html'), name='menu_pages_markup'),
         url(r'^cms/header/branding/markup/$', TemplateView.as_view(template_name='header_branding.html'), name='header_branding_markup'),
         url(r'^cms/header/logo/markup/$', TemplateView.as_view(template_name='header_logo.html'), name='header_logo_markup'),
-    ]
-
-if settings.FEATURES['blog']:
-    urlpatterns += [
-        # Support `taggit_autosuggest` (from `djangocms-blog`)
-        url(r'^taggit_autosuggest/', include('taggit_autosuggest.urls')),
```

Comment:
⚠️ This removes a sub-feature of the CMS blog.

In the new setup, how should this addition to `urls.py` be supported?

_Related to removal of Blog from [`settings.py`](https://github.com/TACC/Core-CMS/pull/341/files#diff-16fd09f645fd22d5389d1711d21583bb9ce289bb09179cb8fa2b6fbf41aac385)._